libqtile/config.py

```diff
@@ -533,11 +535,15 @@ class ScratchPad(Group):
     label : string
         The display name of the ScratchPad group. Defaults to the empty string
         such that the group is hidden in ``GroupList`` widget.
+    exclusive : Boolean
+        Only one of the window among the specified scratchpad should be
+        visible at a time.
     """
-    def __init__(self, name, dropdowns=None, position=sys.maxsize, label=''):
+    def __init__(self, name, dropdowns=None, position=sys.maxsize, label='', exclusive=False):
```

Comment:
This is nice, but I think `exclusive` could be replaced with a more intuitive name. Perhaps `single`?